hbase-server/src/test/java/org/apache/hadoop/hbase/MiniHBaseCluster.java

```diff
@@ -101,16 +102,17 @@ public MiniHBaseCluster(Configuration conf, int numMasters, int numRegionServers
    * @throws InterruptedException
    */
   public MiniHBaseCluster(Configuration conf, int numMasters, int numAlwaysStandByMasters,
-         int numRegionServers, List<Integer> rsPorts, Class<? extends HMaster> masterClass,
-         Class<? extends MiniHBaseCluster.MiniHBaseClusterRegionServer> regionserverClass)
+      int numRegionServers, List<Integer> rsPorts, int numCompactionServers,
```

Comment:
This class is marked as IA.Public, so you need to keep all the old constructors...